packages/babel-preset-mc-app/package.json

```diff
@@ -21,25 +21,27 @@
   "main": "./index.js",
   "module": "./index.js",
   "dependencies": {
-    "@babel/core": "7.3.4",
-    "@babel/plugin-proposal-class-properties": "7.3.4",
+    "@babel/core": "7.4.0",
+    "@babel/plugin-proposal-class-properties": "7.4.0",
     "@babel/plugin-proposal-do-expressions": "7.2.0",
     "@babel/plugin-proposal-export-default-from": "7.2.0",
     "@babel/plugin-proposal-export-namespace-from": "7.2.0",
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.2.0",
-    "@babel/plugin-proposal-object-rest-spread": "7.3.4",
+    "@babel/plugin-proposal-object-rest-spread": "7.4.0",
     "@babel/plugin-proposal-optional-chaining": "7.2.0",
     "@babel/plugin-syntax-dynamic-import": "7.2.0",
-    "@babel/plugin-transform-classes": "7.3.4",
-    "@babel/plugin-transform-destructuring": "7.3.2",
+    "@babel/plugin-transform-classes": "7.4.0",
+    "@babel/plugin-transform-destructuring": "7.4.0",
     "@babel/plugin-transform-react-constant-elements": "7.2.0",
     "@babel/plugin-transform-react-display-name": "7.2.0",
-    "@babel/plugin-transform-regenerator": "7.3.4",
-    "@babel/plugin-transform-runtime": "7.3.4",
-    "@babel/preset-env": "7.3.4",
+    "@babel/plugin-transform-regenerator": "7.4.0",
+    "@babel/plugin-transform-runtime": "7.4.0",
+    "@babel/preset-env": "7.4.2",
     "@babel/preset-react": "7.0.0",
+    "@babel/runtime-corejs2": "7.4.2",
```

Comment:
This is required when using using the `corejs: 2` option in `transform-runtime`

https://babeljs.io/docs/en/next/babel-plugin-transform-runtime#corejs

Reply:
Correct. Same on the other PR.